mooncake-transfer-engine/CMakeLists.txt

```diff
@@ -1,8 +1,82 @@
+cmake_minimum_required(VERSION 3.16)
+project(mooncake-transfer-engine)
+
+if (NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+if (NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 20)
+endif()
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -Wall -Wextra -Wno-unused-parameter -fPIC")
```

Comment:
There should be some detections to avoid multiple definitions in the home CMakeLists.txt.

Reply:
Makes sense, will update.